v1.14/alpine/entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
```

Comment:
Can you add a comment explaining what this entrypoint does - I know you have them lower down but it's useful to have them at the start. Any pre-requisites?

This supports overriding the entire command I believe as well so you can run a shell from the container for example I believe. That's a good entrypoint so just make it obvious so people don't waste time.

Reply:
Again, this isn't my code I copied it from fluentd-docker-image until I can depend on that.